target/ppc/translate/fixedpoint-impl.c.inc

```diff
@@ -81,8 +81,7 @@ static bool do_ldst_quad(DisasContext *ctx, arg_D *a, bool store, bool prefixed)
     if (!prefixed && !(ctx->insns_flags2 & PPC2_LSQ_ISA207)) {
         if (ctx->pr) {
             /* lq and stq were privileged prior to V. 2.07 */
-            gen_priv_exception(ctx, POWERPC_EXCP_PRIV_OPC);
-            return true;
+            REQUIRE_SV(ctx);
         }
```

Comment:
I just noticed that I should've replaced the whole `if` :)
I'll update this in the original branch so you can pull the fix here.

Reply:
The fixup is here: af5e24e86d9cc9b5208c85cfb720735df5846fa6
Please cherry-pick and apply in this commit